### PR TITLE
Hwin16

### PR DESCRIPTION
**Pull request by hwin16**
XMLHandler and schema are updated according to the message Dictionary.

### CODE REVIEW

File: python/lsst/iip/XMLHandler.py [hunk 1]
```diff
@@ -36,19 +35,25 @@ def encodeXML(self, dictValue):
...
-            elif kee != MSG_TYPE:
+
+        for kee, value in pydict.items(): 
+            if kee != "MSG_TYPE" and type(value) != bool:
                 keenode = etree.Element(kee)
                 msg.append(keenode)
                 keenode.text = str(value)
+            elif type(value) == bool: 
+                pydict[kee] = str(value).lower()
+                if msg.find("ACK_BOOL") != None:
```
Comment (timj) [inline python/lsst/iip/XMLHandler.py]: `is not None`

**Comment by timj** [comment]
General comment, can you please remember to schedule the work to fix this package so that it works on python 3? @srp3rd can provide guidance.

**Comment by hwin16** [comment]
Hi Tim, Jim @jbp4 has some questions or plan about migrating to python3. I believe he can be reached at his email.

**Comment by timj** [comment]
Thanks. @jbp4 has contacted me. I shouldn't have made the comment on this PR. I should have made the comment in a different place. Sorry for the confusion.

**Comment by hwin16** [comment]
You are welcome, Tim.

**Pull request merged**